File: client/src/test/java/com/github/goldy1992/mp3player/dagger/components/TestSearchResultActivitySubComponent.java
package com.github.goldy1992.mp3player.dagger.components;

import com.github.goldy1992.mp3player.client.activities.SearchResultActivity;

import dagger.Subcomponent;

@Subcomponent(modules = {})
public interface TestSearchResultActivitySubComponent extends TestMediaActivityCompatComponent  {

    void inject(SearchResultActivity searchResultActivityTest);
}

File: client/testsupport/src/main/java/com/github/goldy1992/mp3player/client/dagger/components/TestSearchResultActivitySubComponent.kt
package com.github.goldy1992.mp3player.client.dagger.components

import com.github.goldy1992.mp3player.client.activities.SearchResultActivity
import dagger.Subcomponent

@Subcomponent(modules = [])
interface TestSearchResultActivitySubComponent : TestMediaActivityCompatComponent {
    fun inject(searchResultActivityTest: SearchResultActivity?)
}